test/github-issues/10190/issue-10190.ts

```diff
@@ -0,0 +1,97 @@
+import "reflect-metadata"
+import {
+    createTestingConnections,
+    closeTestingConnections,
+    reloadTestingDatabases,
+} from "../../utils/test-utils"
+import { DataSource } from "../../../src/data-source/DataSource"
+import { expect } from "chai"
+import { User } from "./entity/User"
+import { Post } from "./entity/Post"
+import { LazyPost } from "./entity/LazyPost"
+import { LazyUser } from "./entity/LazyUser"
+
+describe("github issues > #10190 Cannot save a one-to-many relation", () => {
```

Comment:
We are moving away from creating tests for specific GitHub issues. It would be better to add these tests to the `test/functional/metadata-builder/column-metadata/column-metadata.ts` suite.